CONTRIBUTING.md

```diff
@@ -67,7 +67,7 @@ The following section explains various suggestions and procedures to note during
 ### Prerequisites
 
 * It is strongly recommended that you use Linux distributions systems or OSX for development.
-* Go 1.13.9 or newer installed.
```

Comment:
We're not using at Go 1.18.x only feature. According to our `go.mod` file, Go has to be 1.17.x or higher.